plugin/builtin/configmapgenerator/go.mod

```diff
@@ -8,3 +8,4 @@ require (
 )
 
 replace sigs.k8s.io/kustomize/api v0.6.5 => ../../../api
+replace sigs.k8s.io/kustomize/kyaml v0.9.4 => ../../../kyaml
```

Comment:
to avoid:
>../../../../api/internal/plugins/fnplugin/fnplugin.go:81:4: unknown field 'ContinueOnEmptyResult' in struct literal of type runfn.RunFns

replacement in go.mod from api is ignored. That means that this is the only way.
or to wait for the next kyaml and api release...